packages/bpk-docs/src/pages/AlertPage/AlertPage.js

```diff
@@ -0,0 +1,140 @@
+
+import iosScreenshotDefault from 'react-native-bpk-component-alert/screenshots/ios/default.png';
+import iosCancelable from 'react-native-bpk-component-alert/screenshots/ios/cancelable.png';
+import iosThreeButton from 'react-native-bpk-component-alert/screenshots/ios/three-button.png';
+import androidScreenshotDefault from 'react-native-bpk-component-alert/screenshots/android/default.png';
+import androidCancelable from 'react-native-bpk-component-alert/screenshots/android/cancelable.png';
+import androidThreeButton from 'react-native-bpk-component-alert/screenshots/android/three-button.png';
+
+import IntroBlurb from './../../components/neo/IntroBlurb';
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+import DocsPageWrapper from './../../components/neo/DocsPageWrapper';
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosScreenshotDefault}`,
+        altText: 'iOS Default Alert Component',
+        subText: '(iPhone 8 simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidScreenshotDefault}`,
+        altText: 'Android Default Alert Component',
+        subText: '(Google Pixel emulator)',
+      },
+    ],
+  },
+  {
+    id: 'cancelable',
+    title: 'Cancelable',
+    blurb:
+      'Android alert can optionally be marked as cancelable. ' +
+      'Clicking outside the bounds of a cancelable alert will dismiss it without a callback.',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosCancelable}`,
+        altText: 'iOS Cancelable Alert Component',
+        subText: '(iPhone 8 simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidCancelable}`,
+        altText: 'Android Cancelable Alert Component',
+        subText: '(Google Pixel emulator)',
+      },
+    ],
+  },
+  {
+    id: 'three-button',
+    title: 'Three Buttons',
+    blurb:
+      'iOS alert can accomodate any number of buttons. Each button can ' +
+      'optionally specify a style, which is one of "default", "cancel" or "destructive".',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosThreeButton}`,
+        altText: 'iOS Three button Alert Component',
+        subText: '(iPhone 8 simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidThreeButton}`,
+        altText: 'Android Three button Alert Component',
+        subText: '(Google Pixel emulator)',
+      },
+    ],
+  },
+];
+
+const isNeo = process.env.BPK_NEO;
+
+const AlertSubPage = () => (
+  <DocsPageBuilder
+    title="Alert"
+    blurb={
+      isNeo
```

Comment:
we can remove the `isNeo` logic here since we're not supporting the old version anymore.